README.md

```diff
@@ -53,7 +74,7 @@ player.pause()
 player.quit()
 ```
 
-Playing a file from a URL (courtesy of @jappe999)
+Playing a file from a URL by downloading the network file (courtesy of @jappe999)
```

Comment:
I'm not sure about keeping this example any more now that we can pass URLs to omxplayer. Is there an instance where this would be preferred over instantiating the player with the URL instead?